src/witan/population/england_wales/mye.clj

```diff
@@ -0,0 +1,257 @@
+      :or    {resource-file-name (default-resource-options ::resource-file-name)
+              sheet-name         (default-resource-options ::sheet-name)}}]
+  (with-open [in (-> (or file-path (io/resource resource-file-name))
+                     io/file
+                     io/input-stream)]
+    (-> in
+        fst/input->workbook
+        ((partial some #(when (= sheet-name (.name %)) %)))
+        (ss/sheet->dataset {:n-initial-skip-rows 1
+                            :header-row?         true
+                            :key-fn              keyword
+                            :parser-fn           {:age :int8}
+                            :dataset-name        (or dataset-name
+                                                     (when file-path (str "[" file-path "]" sheet-name))
+                                                     (when resource-file-name (str "[" resource-file-name "]" sheet-name)))}))))
+
+(comment ;; Structure of raw dataset for default options
+  (-> nil
+      ->dataset-raw
+      tc/info
+      (tc/select-columns [:col-name :datatype :n-valid :n-missing :min :max]))
+  ;;=> [myebtablesenglandwales/myebtablesenglandwales20112024.xlsx]MYEB1: descriptive-stats [19 6]:
+  ;;   
+  ;;   |        :col-name | :datatype | :n-valid | :n-missing | :min |    :max |
+  ;;   |------------------|-----------|---------:|-----------:|-----:|--------:|
+  ;;   |       :ladcode23 |   :string |    57876 |          0 |      |         |
+  ;;   |        :laname23 |   :string |    57876 |          0 |      |         |
+  ;;   |         :country |   :string |    57876 |          0 |      |         |
+  ;;   |             :sex |   :string |    57876 |          0 |      |         |
+  ;;   |             :age |     :int8 |    57876 |          0 |  0.0 |    90.0 |
+  ;;   | :population_2011 |  :float64 |    57876 |          0 |  0.0 | 10572.0 |
+  ;;   | :population_2012 |  :float64 |    57876 |          0 |  0.0 | 10521.0 |
+  ;;   | :population_2013 |  :float64 |    57876 |          0 |  0.0 | 10137.0 |
+  ;;   | :population_2014 |  :float64 |    57876 |          0 |  2.0 | 10243.0 |
+  ;;   | :population_2015 |  :float64 |    57876 |          0 |  0.0 | 10384.0 |
+  ;;   | :population_2016 |  :float64 |    57876 |          0 |  0.0 | 10934.0 |
+  ;;   | :population_2017 |  :float64 |    57876 |          0 |  0.0 | 10872.0 |
+  ;;   | :population_2018 |  :float64 |    57876 |          0 |  0.0 | 10969.0 |
+  ;;   | :population_2019 |  :float64 |    57876 |          0 |  0.0 | 11159.0 |
+  ;;   | :population_2020 |  :float64 |    57876 |          0 |  0.0 | 10601.0 |
+  ;;   | :population_2021 |  :float64 |    57876 |          0 |  0.0 | 10771.0 |
+  ;;   | :population_2022 |  :float64 |    57876 |          0 |  0.0 | 11445.0 |
+  ;;   | :population_2023 |  :float64 |    57876 |          0 |  0.0 | 11741.0 |
+  ;;   | :population_2024 |  :float64 |    57876 |          0 |  0.0 | 11874.0 |
+  ;;   
+  
+  :rcf)
+
+(defn ds-raw->ds-by-lad
```

Comment:
I really like how you've done this function. I like the use of `cond->` and how you do things like column selection